.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -80,11 +76,11 @@ runs:
           Write-Verbose "Invoke function with" -Verbose
           Write-Verbose ($functionInput | ConvertTo-Json | Out-String) -Verbose
 
-          $location = Get-AzDeploymentRegion @functionInput
+          $resourceLocation = Get-AzDeploymentRegion @functionInput
```

Comment:
So to double check: this line will later call @ChrisSidebotham function to retrieve one location from available ones minus excluded ones, correct?

Reply:
Correct thiat is picked up in my PR #844